src/styles/utilis/_variables.scss

```diff
@@ -0,0 +1,14 @@
+$background-color-day: #eee;
+$width-day: 100px;
+$height-day: $width-day;
+$width-calendar: (7 * $width-day) + 10px;
+$margin-day: 1px;
+
+$margin: 
+  'mon' 0,
+  'tue' ($width-day * 1)+(1 * $margin-day),
+  'wed' ($width-day * 2)+(2 * $margin-day),
+  'the' ($width-day * 3)+(3 * $margin-day),
+  'fri' ($width-day * 4)+(4 * $margin-day),
+  'sat' ($width-day * 5)+(5 * $margin-day),
+  'sun' ($width-day * 6)+(6 * $margin-day);
```

Comment:
add empty lines in the end of files

Reply:
If you insert empty lines, it gives an error:

Expected no more than 0 empty lines